docs/langfuse.md

```diff
@@ -1,11 +1,28 @@
-# Deploying LangFuse Locally
+# Configuring Kyma Companion for Langfuse
+
+We support the following [configurations](../config/config-example.json) for [Langfuse](https://langfuse.com/docs) in the Kyma Companion:
+
+   ```json
+   "LANGFUSE_HOST": "<Your Host>",
+   "LANGFUSE_SECRET_KEY": "<your secret key>",
+   "LANGFUSE_PUBLIC_KEY": "<your public key>",
+   "LANGFUSE_DEBUG_MODE": "<True/False>",
+   "LANGFUSE_ENABLED": "<True/False>",
+   ```
+
+The `LANGFUSE_HOST` is the host address of your Langfuse instance.
+The `LANGFUSE_PUBLIC_KEY` and `LANGFUSE_SECRET_KEY` can be retrieved from your Langfuse instance. You can learn more about this in the following sections.
+With `LANGFUSE_DEBUG_MODE` we will hide the content of user interactions with the Kyma Companion in the Langfuse traces to prevent privacy issues in any production environment. The default value is `False`.
+Finally, with `LANGFUSE_ENABLED` you can enable or disable the Langfuse integration. The default value is `True`, so you have to set it to `True` if you want to be able to see the content for development work.
```

Comment:
```suggestion
where

* The `LANGFUSE_HOST` is the host address of your Langfuse instance.
* The `LANGFUSE_PUBLIC_KEY` and `LANGFUSE_SECRET_KEY` can be retrieved from your Langfuse instance.
* With `LANGFUSE_DEBUG_MODE` you can hide the content of user interactions with Kyma Companion in Langfuse traces to prevent privacy issues in any production environment. The default value is `False`.
* With `LANGFUSE_ENABLED` you can enable or disable the Langfuse integration. The default value is `True`. Set it to `True` to see the content for development work.

For more information, see the [`config-example.json`](../config/config-example.json).
```